src/test/scala/ch/epfl/bluebrain/nexus/tests/iam/RealmsSpec.scala

```diff
@@ -0,0 +1,175 @@
+              quote("{rev}")        -> s"${rev + 1}",
+              quote("{deprecated}") -> "false"
+            )
+          )
+      }
+    }
+
+    "fetch realm" taggedAs (IamTag, RealmsTag) in {
+      cl.get[Json](s"/realms/$testRealm", Identity.ServiceAccount) {
+        (json, result) =>
+          result.status shouldEqual StatusCodes.OK
+          json.removeKeys(removedKeys:_*) shouldEqual jsonContentOf(
+            "/iam/realms/fetch-response.json",
+            Map(
+              quote("{realm}")     -> testRealmUri,
+              quote("{deltaBase}") -> config.deltaUri.toString(),
+              quote("{rev}")       -> s"${rev + 1}",
+              quote("{label}")     -> testRealm
+            )
+          )
+      }
+    }
+
+    "update realm" taggedAs (IamTag, RealmsTag) in {
+      val body =
+        jsonContentOf(
+          "/iam/realms/update.json",
+          Map(
+            quote("{realm}") -> testRealmUri
+          )
+        )
+
+      cl.put[Json](s"/realms/$testRealm?rev=${rev + 1}", body, Identity.ServiceAccount) {
+        (json, result) =>
+          result.status shouldEqual StatusCodes.OK
+          json.removeKeys(removedKeys: _*) shouldEqual jsonContentOf(
+            "/iam/realms/ref-response.json",
+            Map(
+              quote("{realm}") -> testRealmUri,
+              quote("{deltaBase}") -> config.deltaUri.toString(),
+              quote("{label}") -> testRealm,
+              quote("{rev}") -> s"${rev + 2}",
+              quote("{deprecated}") -> "false"
+            )
+          )
+      }
+    }
+
+    "fetch updated realm" taggedAs (IamTag, RealmsTag) in {
+      cl.get[Json](s"/realms/$testRealm", Identity.ServiceAccount) {
+        (json, result) =>
+          result.status shouldEqual StatusCodes.OK
+          json.removeKeys(removedKeys:_*) shouldEqual jsonContentOf(
+            "/iam/realms/fetch-updated-response.json",
+            Map(
+              quote("{realm}")      -> testRealmUri,
+              quote("{deltaBase}")  -> config.deltaUri.toString(),
+              quote("{rev}")        -> s"${rev + 2}",
+              quote("{label}")      -> testRealm
+            )
+          )
+      }
+    }
+
+    "deprecate realm" taggedAs (IamTag, RealmsTag) in {
+      cl.delete[Json](s"/realms/$testRealm?rev=${rev + 2}", Identity.ServiceAccount) {
+        (json, result) =>
+          result.status shouldEqual StatusCodes.OK
+          json.removeKeys(removedKeys:_*) shouldEqual jsonContentOf(
+            "/iam/realms/ref-response.json",
+            Map(
+              quote("{realm}")      -> testRealmUri,
+              quote("{deltaBase}")  -> config.deltaUri.toString(),
+              quote("{label}")      -> testRealm,
+              quote("{rev}")        -> s"${rev + 3}",
+              quote("{deprecated}") -> "true"
+            )
+          )
+      }
+    }
+
+    "fetch deprecated realm" taggedAs (IamTag, RealmsTag) in {
```

Comment:
I don't think there's a need to duplicate tags... RealmsTag would be sufficient in this example and portable to other resource types as well, right?